src/reflect/intw_avr_test.go

```diff
@@ -0,0 +1,30 @@
+//go:build avr
+
+package reflect_test
+
+import (
+	"reflect"
+	"testing"
+	"unsafe"
+)
+
+// Verify that SliceHeader is the same size as a slice.
+var _ [unsafe.Sizeof([]byte{})]byte = [unsafe.Sizeof(reflect.SliceHeader{})]byte{}
```

Comment:
This still requires testing on AVR to check this. This whole file is never going to be tested because `go test` doesn't work for AVR. So it's dead code.

What I mean is that you put this line in src/reflect/value.go (or similar) so that it is a compile time test and remove these useless tests.